quickstart-services.yml

```diff
@@ -2,6 +2,8 @@ version: '3.7'
 
 volumes:
   alkemio_dev_mysql_db:
+  rabbitmq_data:
+    driver: local
```

Comment:
Keen to understand options that are available for in a production setting

Reply:
That's not very relevant for production setting - in production there are rabbitmq kubectl plugin + rabbitmq cluster operator kubectl plugin that take care for the setup